lib/trackler/track.rb

```diff
@@ -100,9 +100,9 @@ def img(file_path)
     end
 
     def doc_format
-      default_format = 'md'
-      path = File.join(dir, "docs", "*.*")
-      most_popular_format(path) || default_format
+      TOPICS.map { |topic|
+        DocFile.find(basename: topic.upcase, track_dir: dir, image_path: "").extension
+      }.partition(&:to_s).max_by(&:size).first
```

Comment:
I reverted to the old version.

I don't think we should have `doc_format` in the config file. The real fix is to use the filename directly (now that the doc file knows it). I'll take a look at this in exercism/exercism.io